style.css

```diff
@@ -60,30 +64,33 @@ body {
 	margin-top: -25rem
 }
 .header-image {
-	width: 80%;
+	width: 60%;
 	height: 40rem;
+	margin: 5rem auto 0 auto;
+
 }
 .button-border {
 	z-index: 1;
 	background-color: #ffffff;
 	display: flex;
 	justify-content: center;
 	position: absolute;
-	width: 10rem;
-	height: 10rem;
+	width: 9rem;
+	height: 9rem;
 	padding: 3rem auto 0 auto;
 	border-radius: 50%;
-	margin-left: -40rem;
-	margin-top: 4rem;
+	margin-left: -25rem;
+	margin-top: 6rem;
 }
 .play {
 	display: flex;
 	justify-content: center;
 	border: solid;
 	height: 5rem;
 	width: 5rem;
-	margin-top: 2.5rem;
-	padding:0 !important;
+	margin-top: 2rem;
+	margin-left: 2rem;
+	padding: 0 !important;
```

Comment:
Do you know why `!important` is required here?

Reply:
It corrects the shape of the triangle in the play button, without it there is padding in the square. I don't know where that padding came from.

Reply:
Please resolve this so you don't need to use `!important`